### PR TITLE
Update to 5.0-preview7

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: Server/CarChecker.Server.csproj [hunk 1]
```diff
@@ -19,12 +15,14 @@
...
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.3" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="5.0.0-preview.7.20330.10" />
+    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="5.0.0-preview.7.20330.10" />
+    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="5.0.0-preview.7.20330.10" />
+    <PackageReference Include="Microsoft.AspNetCore.Identity.UI" Version="5.0.0-preview.7.20330.10" />
+    <PackageReference Include="Microsoft.AspNetCore.ApiAuthorization.IdentityServer" Version="5.0.0-preview.7.20330.10" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="5.0.0-preview.7.20322.5" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="5.0.0-preview.7.20322.5" />
+    <PackageReference Include="Microsoft.NET.Sdk.Razor" Version="5.0.0-preview.7.20330.10" PrivateAssets="all" />
```
Comment (pranavkm) [inline Server/CarChecker.Server.csproj:25]: The favicon one below is interesting. The file exists in both the server and client project. The PWA manifest hashes the client file, but the server one is in the publish output.

**Review by mkArtakMSFT — APPROVED** [review]
Thanks @pranavkm!

File: Client/CarChecker.Client.csproj [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
     <PackageReference Include="Microsoft.AspNetCore.Components.DataAnnotations.Validation" Version="3.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="3.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Build" Version="3.2.0" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="3.2.0" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Authentication" Version="3.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.0-preview.7.20330.10*" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="5.0.0-preview.7.20330.10" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Authentication" Version="5.0.0-preview.7.20330.10" />
     <PackageReference Include="Microsoft.Extensions.Localization" Version="3.1.3" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.3" />
```
Comment (marek-safar) [inline Client/CarChecker.Client.csproj:18]: Why are these not upgraded to 5.0 as well?

File: Server/CarChecker.Server.csproj [hunk 3]
```diff
@@ -19,12 +15,14 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="3.1.3" />
-    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="3.1.3" />
-    <PackageReference Include="Microsoft.AspNetCore.Identity.UI" Version="3.1.3" />
-    <PackageReference Include="Microsoft.AspNetCore.ApiAuthorization.IdentityServer" Version="3.1.3" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.3" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="5.0.0-preview.7.20330.10" />
```
Comment (SteveSandersonMS) [inline Server/CarChecker.Server.csproj:18]: Should we be thinking of moving this into the shared framework for 5.0?

**Comment by SteveSandersonMS** [comment]
Thanks @pranavkm!

I've merged this into a new branch called `net5`. It's best not to put this in `master` (yet) because it's a public sample people are using to learn about the 3.2.0 release.

You're also now invited as a collaborator so if you want to update other packages (e.g., as per Marek's points above) then you're free to do so. Please don't feel you have to wait for a code review to make further changes on the `net5` branch!

**Pull request merged**